swagger/v1/component_schemas/NpqApplicationAttributes.yml

```diff
@@ -9,11 +9,9 @@ required:
   - teacher_reference_number
   - teacher_reference_number_validated
   - school_urn
-  - headteacher_status
   - eligible_for_funding
-  - funding_choice
-  - course_id
-  - course_name
```

Comment:
Why are we cutting them out? were they never there in the first place, or did something change?

Reply:
so there are 2 things happening here:
- the documentation is out of data.`course_id` and `course_name` no longer exists and now comes under `course_indentifier`
- `headteacher_status` and `headteacher_status` are valid attributes but if you scroll up this is under `required`. and the documentation is incorrect that these fields are not required and may be omitted from the response. This is because `headteacher_status` is only available if they do a headship NPQ and `funding_choice` is only available if they are not `eligible_for_funding`

Reply:
Ah, ok. That makes sense, thanks :)